### ISSUE TITLE
draft task broken in master

### ISSUE THREAD

**Issue by pesterhazy** [hashobject/perun#129]
Probably due to recent refactorings, the `draft` task is currently broken: it doesn't actually cause the drafts not to be rendered (or shown in collections).

Here's a version that works for me:

```clojure
(deftask remove-draft
  []
  (with-pre-wrap fileset
    (let [files (filter identity (pm/get-meta fileset))
          updated-files (->> files
                             (map #(cond-> % (:draft %) (dissoc :content :include-atom :include-rss))))]
      (pm/set-meta fileset updated-files))))
```

**Comment by bhagany** [comment]
Good catch.  I didn't intend to break `draft`, but I also intended to ask about whether or not its deprecated, due to the comment before it - ";; Should be handled by more generic filterer options to other tasks".  Even so, I don't think there's any harm in fixing it.  I should have some time to do that today.

**Comment by bhagany** [comment]
Okay, fixed at #130 - I'd like to give others a bit of time to take a look and object if necessary before I merge it.
